playground/backend/cmd/server/controller.go

```diff
@@ -97,8 +99,30 @@ func (controller *playgroundController) RunCode(ctx context.Context, info *pb.Ru
 		kafkaMockCluster = kafkaMockClusters[0]
 		prepareParams = prepareParamsVal
 	}
+	sources := make([]entity.FileEntity, 0)
+	if len(info.Files) > 0 {
+		for _, file := range info.Files {
+			sources = append(sources, entity.FileEntity{
+				Name:     file.Name,
+				Content:  file.Content,
+				IsMain:   file.IsMain,
+				CntxLine: 1,
```

Comment:
@eantyshev  Am I right that we are setting CntxLine to 1 because it is not used in runners and we do not care about it?